corehq/apps/reports/static/reports/js/case_list.js

```diff
@@ -1,9 +1,26 @@
+hqImport("hqwebapp/js/initial_page_data").get('user_types')
 hqDefine("reports/js/case_list", ['jquery', 'analytix/js/kissmetrix'], function ($, kissAnalytics) {
     $(function () {
         if (document.location.href.match(/reports\/case_list/)) {
             $(document).on('click', 'td.case-name-link', function () {
                 kissAnalytics.track.event("Clicked Case Name in Case List Report");
             });
+            var user_types = hqImport("hqwebapp/js/initial_page_data").get('user_types');
```

Comment:
To echo Jordan's comment above, `"hqwebapp/js/initial_page_data"` should be added to the dependency list in the `hqDefine` call above, along with a parameter, and then you can reference that parameter here. The code you have is correct for modules that aren't yet using requirejs. (And when we talked about analytics I didn't remember to explain the differences between working in requirejs and non-requirejs modules, so apologies for leading you astray, lmk if it'd be helpful to chat about that.)

This is what's causing the test failure `FAIL: corehq.apps.hqwebapp.tests.test_requirejs:TestRequireJS.test_requirejs_disallows_hqimport`

Reply:
https://github.com/dimagi/commcare-hq/pull/22079/commits/428d727fed2307f40b6b6b129fdc3fec446711a2